cg/constants/constants.py

```diff
@@ -306,3 +306,9 @@ class MultiQC(StrEnum):
 
 
 NG_UL_SUFFIX: str = " ng/uL"
+
+
+class SequencingQCStatus(Enum):
+    FAILED = "failed"
+    PASSED = "passed"
+    PENDING = "pending"
```

Comment:
Consider using `auto()` for the values.

Reply:
Resolved in https://github.com/Clinical-Genomics/cg/pull/3279/commits/437ad152d55cb26dcf1a0cd5adb940674800c925